nixos/modules/services/mail/roundcube.nix

```diff
@@ -132,6 +132,7 @@ in
       $config['plugins'] = [${concatMapStringsSep "," (p: "'${p}'") cfg.plugins}];
       $config['des_key'] = file_get_contents('/var/lib/roundcube/des_key');
       $config['mime_types'] = '${pkgs.nginx}/conf/mime.types';
+      $config['temp_dir'] = '/tmp';
```

Comment:
Add a comment explaining that Roundcube uses PHP-FPM which has `PrivateTmp=true` please

Reply:
Added comment.